packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx

```diff
@@ -95,30 +63,26 @@ const DefaultTemplate: Story = (args) => {
 
   return (
     <Box>
-      <Tabs
-        value={value}
-        onChange={handleChange}
-        aria-label="basic tabs example"
-      >
-        <Tab label="Planets" {...a11yProps(0)} />
-        <Tab label="Moons" {...a11yProps(1)} />
-        <Tab
-          label={args.label}
-          {...a11yProps(2)}
-          icon={args.icon}
-          disabled={args.disabled}
-          wrapped={args.wrapped}
-        />
-      </Tabs>
-      <TabPanel value={value} index={0}>
-        Information about Planets
-      </TabPanel>
-      <TabPanel value={value} index={1}>
-        Information about Moons
-      </TabPanel>
-      <TabPanel value={value} index={2}>
-        Information about {args.label}
-      </TabPanel>
+      <TabContext value={value}>
```

Comment:
Re: the TS error, the `value`s here are all position indexes, but they could be any `Tab` identifier.

e.g. one could set the `value` of the first tab to `planet-list` rather than a number.

Reply:
Bad copy-paste on my part from the MUI docs.